data/json/npcs/TALK_SURVIVOR_CHEF.json

```diff
@@ -24,14 +25,68 @@
+      { "text": "Let's trade items.", "topic": "TALK_NONE", "effect": "start_trade" },
+      { "text": "I'm sorry, I gotta go.", "topic": "TALK_DONE" }
+    ]
+  },
+  {
+    "type": "talk_topic",
+    "id": "TALK_NC_SURVIVOR_CHEF_STORY2",
+    "dynamic_line": "Some lunatic covered in a film of goo, black as oil, had fallen through one of the large glass windows.  There were glass shards stuck in it's head and neck.  I thought the poor guy, girl-thing-whatever was dead.  People began to crowd around it, some were taking pictures.",
+    "responses": [
+      { "text": "Horrible. Did you get any pictures yourself?", "topic": "TALK_NC_SURVIVOR_CHEF_STORY3" },
+      { "text": "What did you do before the cataclysm?", "topic": "TALK_NC_SURVIVOR_CHEF_EXPERTISE" },
+      { "text": "Let's trade items.", "topic": "TALK_NONE", "effect": "start_trade" },
+      { "text": "I'm sorry, I gotta go.", "topic": "TALK_DONE" }
+    ]
+  },
+  {
+    "type": "talk_topic",
+    "id": "TALK_NC_SURVIVOR_CHEF_STORY3",
+    "dynamic_line": "No!  I figured the thing dead until it started writhing and spazzing out for a moment.  Everyone jumped back, a few screamed, and one curious stranger stepped in closer, kneeling a little...it attacked him!",
+    "responses": [
+      { "text": "What'd you do?", "topic": "TALK_NC_SURVIVOR_CHEF_STORY4" },
+      { "text": "What did you do before the cataclysm?", "topic": "TALK_NC_SURVIVOR_CHEF_EXPERTISE" },
+      { "text": "Let's trade items.", "topic": "TALK_NONE", "effect": "start_trade" },
+      { "text": "I'm sorry, I gotta go.", "topic": "TALK_DONE" }
+    ]
+  },
+  {
+    "type": "talk_topic",
+    "id": "TALK_NC_SURVIVOR_CHEF_STORY4",
+    "dynamic_line": "I ran to the back of the kitchen and hid as best I could.  People outside were screaming and I could hear them running.  Suddenly I heard more glass shatter and something skitter out of the resturant.  I waited a moment and then went and checked the dining area.  Both the stranger and the thing were gone.  People were running in the streets, some even had guns so I locked all the doors and blocked the windows with what I could and barricaded myself in here.",
+    "responses": [
+      { "text": "Crazy, so you have been here ever since?", "topic": "TALK_NC_SURVIVOR_CHEF_STORY5" },
+      { "text": "What did you do before the cataclysm?", "topic": "TALK_NC_SURVIVOR_CHEF_EXPERTISE" },
+      { "text": "Let's trade items.", "topic": "TALK_NONE", "effect": "start_trade" },
+      { "text": "I'm sorry, I gotta go.", "topic": "TALK_DONE" }
+    ]
+  },
+  {
+    "type": "talk_topic",
+    "id": "TALK_NC_SURVIVOR_CHEF_STORY5",
+    "dynamic_line": "Yeah, it was awhile before it was quite again.  I heard all kinds of sounds; Explosions, jets flying by, helicopters, screaming, and rapid gunfire.  I swear I even heard what sounded like a freaking tank drive by at one time!  I've been hiding here since.",
```

Comment:
```suggestion
    "dynamic_line": "Yeah, it was awhile before it was quiet again.  I heard all kinds of sounds: explosions, jets flying by, helicopters, screaming, and rapid gunfire.  I swear I even heard what sounded like a freaking tank drive by at one time!  I've been hiding here since.",
```